rules/framework.bzl

```diff
@@ -589,6 +666,9 @@ Valid values are:
             default = "framework",
             doc = "The extension of the bundle, defaults to \"framework\".",
         ),
+        "environment_plist": attr.label(
+            allow_single_file = True,
```

Comment:
Consider linking to the rules_apple doc on this one - I didn't know what it was used for.